manifests/install.pp

```diff
@@ -55,5 +58,13 @@
       tag    => 'nodesource_repo',
     }
   }
-}
 
+  file { 'root_npmrc':
```

Comment:
If the docs read that the whole file is managed, how do you think we should do that? I'm thinking a concat resource.